adjusters.md

```diff
@@ -19,39 +17,31 @@ Event::on(OrderAdjustments::class, OrderAdjustments::EVENT_REGISTER_ORDER_ADJUST
 });
 ```
 
-The order of the adjustments within the types array is important, as it is the order they will be run through on the order.
-
-You could have a project level event listener, that could reorder these adjusters, and not append any new ones.
+The order of the adjustments within the types array is important, because its adjusters will be called in exactly that sequence when an order is calculated. Because of this, you could have a project-level event listener that could exist solely to reorder these adjusters without appending any new ones.
```

Comment:
"when an order is *calculated*"

The function on the order model is called `->recalculate()`. Might be good to figure out which word to use. Also would probably be good to define what recalculation is and when it happens (this page or a link). e.g:


Recalculation

While the order is a cart, the order recalculates every time you interact with it. For example:

- Add something to the cart, recalculate, save cart, and return cart to the customer on the front end.
- Save cart address, recalculate, save cart, and return cart to the customer on the front end.

Every time the order recalculates, it drops all adjustment models from the order and runs through all registered adjusters again to add back the calculated adjustment models.